pegasus/sites.v3/code.org/public/css/container-responsive-full-width.css

```diff
@@ -31,7 +31,7 @@
 /* This is better for accessibility */
 section {
   width: 100%;
-  padding: 3em 1em;
+  padding: 4em 2em;
```

Comment:
Increases section whitespace on the top for all screen sizes, and on the sides for tablet and mobile.